### DISCUSSION TITLE
Flatpack instalation

### DISCUSSION THREAD

**Discussion by DinoIsanovic** [OpenBoard-org/OpenBoard#1144] [Q&A] [answered]
Any chance for Flatpack version ??

**Comment by letsfindaway** [comment, marked as answer]
Any chance that people just read the [README](https://github.com/OpenBoard-org/OpenBoard#communnity-driven-packages)?
Any chance that people read the [Downloads](https://github.com/OpenBoard-org/OpenBoard/wiki/Downloads) wiki page?
Any chance that people search in the already existing issues?
Any chance that people just have a look at [Flathub](https://flathub.org/apps/ch.openboard.OpenBoard)?

This question was asked and answered so many times before.

**Reply by DinoIsanovic** [reply]
My fault, I was lazy.
But thx I just jumped from Ubuntu to Manjaro. 
Due last update with strange bug.
Update!
I found today Openboard in Manjaro package manager, he wasn't there yesterday.
Strange but nice :) 
And Openboard works much better under Manjaro, I have a big cross for my graphic pen.
